### PR TITLE
Spellchecks, fixes, and m- Wait? Guns, again?

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

This PR introduces a couple of fixes, spellchecks and balance changes for the sake of consistency.

## Why It's Good For The Game

Fixes stuff, makes stuff easier to read, consistency with changes applied in #6901 

## Changelog
:cl:
balance: Applied changes from PR #6901 to remaining firearms.
fix: DZR-9 was described as a 9mm PDW, but had its caliber set to 10MM in its code.
spellcheck: Changed certain descriptions to reflect a fix.
spellcheck: Magazine names now match those applied in PR #6901 
/:cl:

### CODE REVIEW

**Comment by Draganfrukts** [comment]
> You currently have a negative Fix/Feature pull request delta of -6. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

### Shut yo bitch ass up!

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/glock.dmi'
 	icon_state = "glock"
 	w_class = WEIGHT_CLASS_NORMAL
-	mag_type = /obj/item/ammo_box/magazine/multi_sprite/g17
+	mag_type = /obj/item/ammo_box/magazine/multi_sprite/g17, /obj/item/ammo_box/magazine/multi_sprite/g18
```
Comment (Draganfrukts) [inline modular_skyrat/modules/sec_haul/code/guns/guns.dm]:
```suggestion
	mag_type = /obj/item/ammo_box/magazine/multi_sprite/g17 | /obj/item/ammo_box/magazine/multi_sprite/g18
```

**Comment by ZephyrTFA** [comment]
That's not how the mag_type var works

**Comment by Draganfrukts** [comment]
I'll fix this when I have the energy to


(7 months)

**Comment by ZephyrTFA** [comment]
> I'll fix this when I have the energy to
> 
> (7 months)

Understandable

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm [hunk 2]
```diff
@@ -25,7 +25,7 @@
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/glock.dmi'
 	icon_state = "glock"
 	w_class = WEIGHT_CLASS_NORMAL
-	mag_type = /obj/item/ammo_box/magazine/multi_sprite/g17
+	mag_type = /obj/item/ammo_box/magazine/multi_sprite/g17 | /obj/item/ammo_box/magazine/multi_sprite/g18
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/sec_haul/code/guns/guns.dm]: This isn't valid, guns don't support two types of mag.

Comment (ZephyrTFA) [reply modular_skyrat/modules/sec_haul/code/guns/guns.dm]: Already toooold them

**Pull request merged**